gatsby-config.js

```diff
@@ -12,19 +12,22 @@
 module.exports = {
     pathPrefix: process.env.PATH_PREFIX || '/client-sdks/previous-versions/',
     siteMetadata: {
-        versions: [
+        pages: [
             {
-                title: 'Latest version',
-                path: '/client-sdks/documentation',
-
-              },
-              {
-                title: 'Previous versions',
-                path: '/client-sdks/previous-versions/documentation',
-                selected: true
-              }
-          ],
-        pages: [{
+                title: "Versions",
+                menu: [
+                    {
+                        title: "Latest version",
+                        path: "https://developer.adobe.com/client-sdks/home/",
```

Comment:
should this be just /client-sdks/home or is this link no longer pointing to the current sdk versions site?

Reply:
I think that would work as well. I'll change it to that.

Reply:
I checked, and I don't think the relative link works - so I've changed it back to an absolute link.